play-json/shared/src/main/scala/play/api/libs/json/JsLookup.scala

```diff
@@ -125,23 +125,14 @@ sealed trait JsLookupResult extends Any with JsReadable {
   /**
```

Comment:
Please also moves the `JsLookupResult` sealed family in a separate `JsLookupResult.scala`

Reply:
Done in 4bdf67c